### DISCUSSION TITLE
Is that the file of initial structure in extrapolation should be 'POSCAR' type?

### DISCUSSION THREAD

**Discussion by caohao001** [deepmodeling/dpgen#441] [Q&A]
Hello，
The ab initio data for training model is the .xyz type which the index of atoms is not arranged according to the elements. Is that mean the extrapolation step cannot proceed for the sys_configs should be the POSCAR files?
Thanks!

**Comment by AnguseZhang** [comment]
First, you should see https://github.com/deepmodeling/dpdata/ . `DP-GEN` uses `dpdata` to manipulate the formats of different softwares.  You can try to use `dpdata` to load your `.xyz` file with format = "quip/gap/xyz", and see whether it can works.

Second, if it works, you can specify the file format by specifying  `"sys_format" : "quip/gap/xyz"` in `param.json`, see https://github.com/deepmodeling/dpgen/blob/5b54014617ac5cf4f17ee0164cfba8ebad81b28c/examples/run/dp1.x-gromacs-gaussian/param.json#L37 . 

Last, `.xyz` actually is not a well-defined format, and it may cause problems. Instead, `POSCAR` is more stable, according to my experience. 

Yuzhi

**Reply by caohao001** [reply]
Thanks for your reply and suggestion.
